package.json

```diff
@@ -50,7 +50,7 @@
     },
     "devDependencies": {
         "@types/mocha": "^2.2.43",
-        "@types/node": "^8.0.32",
+        "@types/node": "^7.0.56",
```

Comment:
The only versions of @types/node for node.js version 7.x are of `7.0.y` format, so this pulls the latest (which is marked in the source as targeting the major 7.x version)